Source/IO/Conversions/KTROOTData.hh

```diff
@@ -67,6 +70,87 @@ namespace Katydid
             ClassDef(TProcessedTrackData, 1);
     };
 
+    //***********************
```

Comment:
As I mentioned in Slack, all classes defined here need to move to Cicada, and the Load() and Unload() functions go in KT2ROOT.  Please put them in their own class files there, as opposed to adding to the _ROOTData files.